nextstrain/cli/runner/conda.py

```diff
@@ -579,6 +580,21 @@ def package_name(spec: str) -> str:
     return PackageSpec.parse(spec).name
 
 
+def latest_package_label_version(channel: str, package: str, label: str) -> str:
+    response = requests.get(f"https://api.anaconda.org/package/{urlquote(channel)}/{urlquote(package)}/files")
+    response.raise_for_status()
+
+    def file_upload_datetime(file):
+        # Python 3.6 does not recognize colon as separator for %z directive
+        upload_time = file["upload_time"].replace("+00:00", "Z")
+        upload_time_format = '%Y-%m-%d %H:%M:%S.%fZ'
+        return datetime.datetime.strptime(upload_time, upload_time_format)
```

Comment:
Hmm, yeah. We probably won't change the format of the version string in [`conda-base/devel/generate-version`](https://github.com/nextstrain/conda-base/blob/560e502589bf237eb66b94428f58e8b9528497a4/devel/generate-version)?

Reply:
Unlikely, but even non-date-based versions have a well-defined ordering. The future breakage that I think is more likely is one-off rebuilds of previous versions (e.g. taking some YYYYMMDD… build 0 release and rebuilding it from the same source as build 1 to fix some error; it then gets most recent upload time but isn't the highest version).

Reply:
OH actually, this would have to be a string comparison to account for other packages that may have a different version formats.

Reply:
Updated in https://github.com/nextstrain/cli/pull/280/commits/ff34c77490d0f0d38d44d9dd51722f0c7ca6acbb

Reply:
Ah, we should use version objects so that we get version-aware ordering which can handle both x.y.z style versions and our string-style versions.

You can see them in use already in `nextstrain.cli.util.new_version_available()`.

Reply:
Updated in https://github.com/nextstrain/cli/pull/280/commits/28608ded51e9099989e2de72e0849b32160633cf